test/bot/bot.jl

```diff
@@ -319,35 +319,37 @@ bottestdir = GoodPath(@__DIR__)
 
 
     using Pkg
+    package_rootpath = Symbol[]
     for (i, package_name) in enumerate(["TestPackage1", "TestPackage2", "TestPackage3", "TestPackage4"])
         Pkg.develop(PackageSpec(path=joinpath(bottestdir,"$package_name.jl")))
-        @eval $(Symbol("package_rootpath", i)) = goodjoinpath(bottestdir,"$($package_name).jl")
+        push!(package_rootpath, Symbol(goodjoinpath(bottestdir,"$package_name.jl")))
```

Comment:
I'm really against creating variable names like this. It was hard for me to find where they were defined.

Reply:
Sure! This is better.